collect_app/src/main/java/org/odk/collect/android/widgets/GeoPointWidget.java

```diff
@@ -303,14 +303,10 @@ public void setBinaryData(Object answer) {
 		if (!s.equals("") || s == null){
 			mStringAnswer.setText(s);
 			String[] sa = s.split(" ");
-			mAnswerDisplay.setText(getContext().getString(R.string.latitude) + ": "
-					+ formatGps(Double.parseDouble(sa[0]), "lat") + "\n"
-					+ getContext().getString(R.string.longitude) + ": "
-					+ formatGps(Double.parseDouble(sa[1]), "lon") + "\n"
-					+ getContext().getString(R.string.altitude) + ": "
-					+ truncateDouble(sa[2]) + "m\n"
-					+ getContext().getString(R.string.accuracy) + ": "
-					+ truncateDouble(sa[3]) + "m");
+			mAnswerDisplay.setText(String.format(getContext().getString(R.string.gps_result),
+					formatGps(Double.parseDouble(sa[0]), "lat"),
```

Comment:
Ah, just kidding, those are used to determine the cardinal direction (N / S or E / W). What needs to be localized are the cardinal direction strings in `formatGps`. That will require some refactoring and Lint doesn't see that those are eventually user-facing strings so I've filed at #329